configure.ac

```diff
@@ -146,9 +146,9 @@ AC_CACHE_SAVE
 # during 'make distcheck'.
```

Comment:
Also need to remove https://github.com/endlessm/eos-metrics/blob/master/Makefile.am#L47 and the following line.